VideoLocker/src/main/java/org/edx/mobile/loader/CoursesAsyncLoader.java

```diff
@@ -127,5 +127,4 @@ public void deliverResult(AsyncTaskResult<List<EnrolledCoursesResponse>> data) {
     private void releaseResources(AsyncTaskResult<List<EnrolledCoursesResponse>> data) {
 
     }
-
-}
+}
```

Comment:
Yeah, not sure how to get rid of that. I even deleted the line in Android
Studio and entered the ‘}’ character again. It didn’t seem to make a
difference.

On Fri, Jul 1, 2016 at 2:31 PM, Christopher Lee <notifications@github.com>
wrote:

> In VideoLocker/src/main/java/org/edx/mobile/loader/CoursesAsyncLoader.java
> <https://github.com/edx/edx-app-android/pull/762#discussion_r69339364>:
>
> > @@ -127,5 +127,4 @@ public void deliverResult(AsyncTaskResult<List<EnrolledCoursesResponse>> data) {
> >      private void releaseResources(AsyncTaskResult<List<EnrolledCoursesResponse>> data) {
> >
> >      }
> > -
> > -}
> > +}
>
> newline
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/edx/edx-app-android/pull/762/files/ea9b07b4158faf859172c492dbcea9e5e78e15a1#r69339364>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AR10T9-liTb8sH2MV5oGOg_f4oij9eXkks5qRV0DgaJpZM4JDSLB>
> .
>



-- 
Regards,

Marc DiNino
Android Engineer
EDX